data/data.js

```diff
@@ -1,15 +1,17 @@
-var $chromatic = ["A", "A♭", "A♯", "B", "B♭", "B♯", "C", "C♭", "C♯", "D", "D♭", "D♯", "E", "E♭", "E♯", "F", "F♭", "F♯", "G", "G♭", "G♯"];
+// musical assets for the site to use 
```

Comment:
Hi Kenny, thanks for your comments!

I will get to the fixes likely next week after my tech challenge is complete.

On the issue of indentation, I use the prettier extension in vscode which set the indentation automatically. If you're not familiar with it, It's an auto-linter of sorts that keeps files consistent and lets me focus on other things than spacing, indention, bracket alignment, etc. I find it useful. It's my understanding that using is standard practice in a lot of tech workplaces.  Here's it's [vscode github](https://github.com/prettier/prettier-vscode).  If you really are against using it, please let me know.  If you have another linting service that you prefer, let me know too.

On the issue of #13, do you first agree that it's enharmonically equivalent to a b7? 
For eg, on a Cmin#13 chord, the pitches would be C Eb G A#.  Where Bb is = A#, so this would be more commonly known as a Cmin7 chord. In something like a C9#13, the pitches would be C E G Bb D A#.  This would be just a doubling of the b7, so the #13 doesn't add additional chord quality information.

In both cases, I find the #13 unuseful because it doesn't add additional information not already contained in a b7.

If you do not agree with this assessment, please let me know why not.

If you agree that it is equivalent to a b7 and want still want to use it, why is that? Or, what value does it add?  

As an aside, In my 20+ years of learning jazz, including during my undergrad and grad work, I've not encountered it. That doesn't mean it doesn't exist, but it means that if it does exist, it's quite rare. In a quick google search, I can't find any examples of it. If you know of examples, please share them!  I'd love to see it in use.

More comments below.  Thanks!

Reply:
A #13 _note_ would not be enharmonically equivalent to a b7, but they don't function the same. 9, 11, and 13 chords are dominant extensions and include those below them.

C7 = C, E, G, Bb (1, 3, 5, b7)
C9 = C, E, G, Bb, D (1, 3, 5, b7, 9)
C11 = C, E, G, Bb, D, F (1, 3, 5, b7, 9, 11)
C13 = C, E, G, Bb, D, F, A (1, 3, 5, b7, 9, 11, 13)

Maybe a #13 would be super rare, but it would functionally be different from a dominant 7th chord because...

C7 = C, E, G, Bb (1, 3, 5, b7)
C13 = C, E, G, Bb, D, F, A# (1, 3, 5, b7, 9, 11, #13)

...and I guess I've just made your point. :D Okay, you win. Technically, it's different, but now that I write that out, I see that the 13th chord, which includes the b7, means that a #13 would include a redundant/enharmonic #13 

Thanks for helping me work through that. :)

Reply:
...however. 👍 

I'm going to fix this in a separate PR, so go ahead and undo this change for the sake of your PR, please. Thanks! (Or you might be able to rebase after I commit mine.) I'm doing this because I think there are a couple references to this that I need to fix.